app/assets/images/payment_icons/payconiq.svg

```diff
@@ -0,0 +1 @@
9,0.2851l-6.007771,4.3183l-4.829072,3.471178c-0.080078,0.064053 -0.163649,0.124691 -0.250849,0.181292c-0.1583,0.10235 -0.32324,0.185751 -0.491499,0.255629c-1.226482,0.511261 -2.664211,0.168182 -3.526241,-0.862019c-0.079269,-0.09474 -0.15522,-0.193279 -0.22404,-0.29952l-3.1592,2.047369c0.992559,1.531582 2.52211,2.584942 4.30664,2.966072c0.11895,0.025429 0.238141,0.047449 0.357331,0.066479c0.11636,0.018459 0.23288,0.033852 0.349308,0.046242c0.00235,0.000319 0.0047,0.000729 0.007051,0.000969c0.238138,0.025181 0.476358,0.037809 0.71393,0.037809c1.311739,0 2.60858,-0.384609 3.786469,-1.14963c0.00316,-0.002029 0.006161,-0.004299 0.009399,-0.006401l0.387852,-0.27879l6.01328,-4.32291l5.076603,-3.649389c0.07692,-0.04996 0.155945,-0.09474 0.235947,-0.13717c0.023239,-0.01231 0.046722,-0.02381 0.070202,-0.03547c0.06284,-0.03133 0.1264,-0.06024 0.190769,-0.08704c0.020493,-0.0085 0.040649,-0.01757 0.061302,-0.02567c0.084858,-0.033031 0.170769,-0.06283 0.257889,-0.08818c0.007938,-0.00226 0.016121,-0.003881 0.024048,-0.00615c0.080238,-0.02259 0.161301,-0.041301 0.243004,-0.05709c0.019836,-0.003881 0.039665,-0.007441 0.059586,-0.010931c0.08049,-0.01392 0.161381,-0.02494 0.242844,-0.032379c0.009708,-0.0009 0.019348,-0.00235 0.029148,-0.00316c0.08186,-0.006639 0.164051,-0.01085 0.246559,-0.01085c0.008823,0 0.017731,0.00137 0.02655,0.001451c0.059361,0.00049 0.11887,0.00543 0.178303,0.009399c0.043728,0.002911 0.087288,0.00348 0.131096,0.008181c0.103642,0.01133 0.207283,0.027359 0.310524,0.04931c0.100159,0.02137 0.198627,0.04753 0.294899,0.078219c0.090927,0.0289 0.1796,0.062591 0.26664,0.09951c0.027859,0.01182 0.054581,0.02575 0.08202,0.0383c0.058468,0.02688 0.1166,0.054411 0.173038,0.084781c0.031181,0.01676 0.061218,0.03498 0.09166,0.052791c0.051182,0.029959 0.101784,0.06073 0.151093,0.093519c0.030289,0.020161 0.059921,0.04122 0.089478,0.06251c0.047932,0.0345 0.09473,0.07021 0.140564,0.10745c0.027534,0.02235 0.054897,0.044861 0.081707,0.06818c0.046879,0.04089 0.09206,0.08372 0.136509,0.12753c0.022762,0.022429 0.046001,0.04437 0.0681,0.06761c0.050041,0.05247 0.097649,0.10761 0.144131,0.16389c0.014011,0.01692 0.028912,0.03295 0.042591,0.05012c0.060081,0.07619 0.117649,0.15482 0.171097,0.23717l3.159271,-2.047131l-0.000088,-0.000239z" fill="#C40041"/><path d="m8.798023,9.516718l-2.04834,-3.15821c-3.16073,2.04939 -4.065101,6.28761 -2.016191,9.44785l3.15919,-2.047369c-0.17247,-0.266071 -0.2983,-0.549152 -0.3796,-0.839512c-0.08129,-0.290369 -0.117981,-0.588019 -0.11223,-0.88348c0.01919,-0.984779 0.51045,-1.94389 1.39717,-2.519279m21.18457,0.90682c0.445503,0.68705 0.597,1.50583 0.425751,2.30672c-0.170853,0.80203 -0.64349,1.488021 -1.3311,1.93352l2.04834,3.16025c1.531101,-0.99304 2.584461,-2.522669 2.965103,-4.30769c0.053925,-0.252642 0.091827,-0.506161 0.117165,-0.759521c0.017654,-0.17717 0.029072,-0.354409 0.032875,-0.53109c0.000408,-0.01684 0.001709,-0.03369 0.001945,-0.05045c0.019432,-1.33822 -0.356117,-2.651979 -1.100727,-3.79984l-3.159351,2.04713" fill="#D62560"/></svg>
```

Comment:
From the [contributing](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#appearance) document, would it be possible to have the border should black (hex color #000) with a 0.07% opacity?

Here is how it looks to compare to other icons: https://screenshot.click/09-22-dch31-t0pyj.mp4